apple/Omnivore.xcodeproj/project.pbxproj

```diff
@@ -1439,7 +1439,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = QJF2XZ86HB;
+				DEVELOPMENT_TEAM = MHM9HW2S8F;
```

Comment:
Can you remove this file from the PR? Apple just does this automatically.

Reply:
Yes, sorry I didn't notice it